syncopy/nwanalysis/connectivity_analysis.py

```diff
@@ -240,11 +253,18 @@ def connectivityanalysis(data, method="coh", keeptrials=False, output="abs",
             log_dict["tapsmofrq"] = tapsmofrq
 
         check_effective_parameters(ST_CrossSpectra, defaults, lcls)
+
+        if method == 'granger':
+            demean_taper = True
+        else:
+            demean_taper = False
+
         # parallel computation over trials
         st_compRoutine = ST_CrossSpectra(samplerate=data.samplerate,
                                          nSamples=nSamples,
                                          taper=taper,
                                          taper_opt=taper_opt,
+                                         demean_taper=demean_taper,
```

Comment:
Nit-picky comment: we could shorten this to 
```
st_compRoutine = ST_CrossSpectra(samplerate=data.samplerate, ..., demean_taper=method=='granger', ...)
```

Reply:
uff, never seen that before it would work of course..